iec_api/models/response_descriptor.py

```diff
@@ -11,8 +11,8 @@ class ResponseDescriptor(DataClassDictMixin):
     """Response Descriptor"""
 
     is_success: bool = field(metadata=field_options(alias="isSuccess"))
-    code: Optional[str]
-    description: Optional[str]
+    code: Optional[str] = None
```

Comment:
No reason for setting default values, they'll be `None` by default

Reply:
I had some errors without this added line. for example: `errorMessage": "Field \"response_descriptor\" of type ResponseDescriptor in ResponseWithDescriptor has invalid value {'isSuccess': True, 'code': '00'}",`

Reply:
When trying to parse the response:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 6, in __mashumaro_from_dict__
mashumaro.exceptions.MissingField: Field "data" of type Optional[Any] is missing in ResponseWithDescriptor instance
```

This means that `data` field (and not `code`) can be None.
Which you already added.